templates/terraform/post_create/org_security_policy.go.erb

```diff
@@ -0,0 +1,41 @@
+parent := d.Get("parent").(string)
+err = computeOrgOperationWaitTime(
+    config, res, parent, "Creating OrganizationSecurityPolicy",
+    d.Timeout(schema.TimeoutCreate))
+
+if err != nil {
+  // The resource didn't actually create
+  d.SetId("")
+  return fmt.Errorf("Error waiting to create OrganizationSecurityPolicy: %s", err)
+}
+
+// We need to check the policy ID in the final operation result
+operationName, ok := res["name"]
+if !ok {
+  return fmt.Errorf("Operation didn't contain name. Cannot get targetId.")
+}
+
+operationUrl := fmt.Sprintf("{{ComputeBasePath}}locations/global/operations/%s?parentId={{parent}}", operationName.(string))
```

Comment:
Instead of polling the operation and then doing an extra GET here, could you have computeOrgOperationWaitTime return the value of the final operation? Something like https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/operation.go.erb#L52

Reply:
Thanks for pointing me to it, much better with WithResponse!